warehouse/templates/manage/roles.html

```diff
@@ -50,6 +50,11 @@ <h2>Collaborators</h2>
       <tr class="collaborator">
         <td class="table__user">
           <a href="{{ request.route_path('accounts.profile', username=role.user.username) }}">
+            {% set alt = "Avatar for {} from gravatar.com".format(role.user.username) %}
+            <a href="{{ request.route_path('accounts.profile', username=role.user.username) }}" class="tooltipped tooltipped-e" aria-label="{{ role.user.username}}" data-original-label="{{ role.user.username}}">
+              <img src="{{ gravatar(role.user.email, size=50) }}" height="50" width="50" alt="{{ alt }}" title="{{ alt }}">
+            </a>
+            <a href="{{ request.route_path('accounts.profile', username=role.user.username) }}">
```

Comment:
Looks like we're missing the closing tag for this `<a>`.

Reply:
I will add it.